components/ProgressCard.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import ProgressCard from './ProgressCard'
+
+describe('ProgressCard component', () => {
+  it('should render correct lesson progress for 0% completion', () => {
+    const { container } = render(<ProgressCard progressCount={0} />)
+    expect(container).toMatchSnapshot()
```

Comment:
No need to check against new snapshot every time. Testing library has more advanced [queries](https://testing-library.com/docs/queries/about/). For example, you can check if `You're off to a great start!` is present on the screen:

` expect(screen.getByText("You're off to a great start!")).toBeTruthy()`

Or if your component changes its state over time. 
` await waitFor(() => screen.getByText('You're off to a great start!'))`

It's just for your information. Your PR is fine as it is.

Reply:
Thank you for explaining this. Still very fresh using Jest.